views/html/rankings/holes.html

```diff
@@ -6,42 +6,56 @@
 <main id=two-pane>
     {{ template "nav" . }}
 
-    <table class="nowrap-second sticky">
-        <thead>
-            <tr>
-                <th>#
-                <th>Golfer
-                {{ if eq .Data.HoleID "all" }}<th class=right>Holes{{ end }}
-                <th class=right>Points
-                <th class=right>{{ title .Data.Scoring }}
-                <th class=right>Time
-        <tbody>
-    {{ $name := "" }}
-    {{ with .Golfer }}{{ $name = .Name }}{{ end }}
+        <table class="nowrap-second sticky">
```

Comment:
Can we avoid this indentation, it doesn't make sense as table is the direct descendant of main and it makes the diff harder to follow.